android/src/com/mapswithme/maps/routing/NavigationController.java

```diff
@@ -59,7 +59,7 @@
 
   private final SearchWheel mSearchWheel;
 
-  private boolean mShowTimeLeft = true;
+  private static boolean mShowTimeLeft = true;
```

Comment:
Hey @fijemax , now it's much better. There are some notes and I put all them in comments, please read it.